doajtest/unit/test_journalcsv.py

```diff
@@ -1,4 +1,4 @@
-import time
+import time, sys
```

Comment:
This addition doesn't seem to be used in this test. We can just checkout the old version of this file by the look of the diff.